draft-ietf-tls-tls13.md

```diff
@@ -775,28 +775,28 @@ and potentially some other extensions.
 
 The server processes the ClientHello and determines the appropriate
 cryptographic parameters for the connection. It then responds with its
-own ServerHello which indicates the negotiated connection
+own ServerHello, which indicates the negotiated connection
 parameters. [{{server-hello}}]. The combination of the ClientHello
 and the ServerHello determines the shared keys. If (EC)DHE
 key establishment is in use, then the ServerHello
-will contain a "key_share" extension with the server's ephemeral
+contains a "key_share" extension with the server's ephemeral
 Diffie-Hellman share which MUST be in the same group as one of the
 client's shares. If PSK key establishment is
-in use, then the ServerHello will contain a "pre_shared_key"
+in use, then the ServerHello contains a "pre_shared_key"
 extension indicating which of the client's offered PSKs was selected.
 Note that implementations can use (EC)DHE and PSK together, in which
 case both extensions will be supplied.
 
 The server then sends two messages to establish the Server Parameters:
 
 EncryptedExtensions.
-: responses to any extensions which are not required in order to
+: responses to any extensions that are not required to
   determine the cryptographic parameters. [{{encrypted-extensions}}]
 
 CertificateRequest.
-: if certificate-based client authentication is desired, the
-  desired parameters for that certificate. This message will
-  be omitted if client authentication is not desired.
+: if certificate-based client authentication is requested, the
```

Comment:
I think "desired" is the right word here, because it's this message which tells you to do it.